### DISCUSSION TITLE
iOS 17.6.1: Pin input SMS auto fill not working

### DISCUSSION THREAD

**Discussion by sinchang** [chakra-ui/zag#2119] [General]
# 🐛 Bug report

## 💻 Link to reproduction

CodeSandbox reproduction: https://your-codesandbox-link-goes-here.io

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      |     0.78.0       |
| Browser          |     17.6.1       |
| Operating System |     iOS       |

## 📝 Additional information

The screenshot shows the user received code 534418, but entered 534415.  Our QA confirmed this error consistently replaces the final digit with the first. 

It works well on other iOS versions.

**Comment by segunadebayo** [comment]
This is now fixed
